library/spdm_requester_lib/libspdm_req_get_csr.c

```diff
@@ -198,11 +198,19 @@ static libspdm_return_t libspdm_try_get_csr(libspdm_context_t *spdm_context,
         goto receive_done;
     }
 
+    if (spdm_response_size < sizeof(spdm_csr_response_t)) {
```

Comment:
This can replace the check at line 174.

Reply:
I think it is better to split the two check. 
Because the response may be error message,  the error message size is `sizeof(spdm_message_header_t)`.
Thanks.

Reply:
Ah yeah that's right. I always forget about `ERROR`.